src/test/resources/pages/instructorCourseEditObserver.html

```diff
@@ -715,7 +715,7 @@
   <br>
   <br>
   <div class="align-center">
-    <input class="btn btn-primary" id="btnShowNewInstructorForm" onclick="showNewInstructorForm()" type="button" value="Add New Instructor">
+    <input class="btn btn-primary" disabled="" id="btnShowNewInstructorForm" onclick="showNewInstructorForm()" type="button" value="Add New Instructor">
```

Comment:
Since an instructor with "Observer" cannot modify instructors, the "Add New Instructor" button is disabled in the InstructorCourseEditPage.